src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -202,14 +202,23 @@ Let's say your organization recently updated images on the site’s home page on
     </Collapser>
 </CollapserGroup>
 
-## What's next? [#next]
+## Get more context with session traces [#session-traces]
+
+New Relic captures data about a user's session on a given page. Session traces are randomly sampled and recorded in New Relic, letting you view how a user experienced page load timing issues, JavaScript events, or other errors. 
```

Comment:
This sentence is good in terms of info, but I would suggest maybe personalizing it a bit and removing "New Relic" directly if possible. Stuff like "We regularly capture data" instead of "New Relic captures data" as an example.